samples/bookinfo/platform/kube/rbac/upgradeToClusterRbacConfig.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+set -e
+set -u
+
+# This script converts RbacConfig to ClusterRbacConfig and deletes the RbacConfig after ClusterRbacConfig
```

Comment:
Would be nicer to add some background for this. Also, does this RbacConfig is the *broken* one (i.e the one in namespace scoped) or the correct one (cluster-scoped) but need to migrate. If for the former, is this necessary as I would expect you cannot get the policy from pilot anyway.

Reply:
Updated the comment.
It doesn't matter is this RbacConfig the broken one or the correct one, the code get the RbacConfig from all namespaces. If there is more than 1 returned, it will actually pick one with name "default" randomly.